index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Martin Karugaba</title>
+    <link rel="stylesheet" href="../dist/output.css" />
+  </head>
+  <body class="font-poppins">
+    <!--* navbar -->
+    <nav
+      class="h-14 w-full px-5 py-4 bg-slate-50 flex justify-between items-center"
+    >
+      <!--* logo -->
+      <div
+        class="text-light-blue leading-3 gap-y-0 flex flex-col justify-between items-start cursor-pointer font-semibold"
+      >
+        <p class="text-2xl">Martin</p>
+        <p class="-mt-[10px] text-2xl">Karugaba</p>
+      </div>
+
+      <!--* nav links -->
+      <div
+        class="w-[26px] h-[30px] border-white flex flex-col justify-center items-center cursor-pointer"
+      >
+        <!--* hamburger -->
+        <div class="h-[3px] w-[100%] bg-light-blue rounded-sm"></div>
+        <div
+          class="h-[3px] w-[100%] bg-light-blue my-[6px] rounded-sm"
+        ></div>
+        <div class="h-[3px] w-[100%] bg-light-blue rounded-sm"></div>
+      </div>
+    </nav>
+    <!--* end navbar -->
+
+    <!--* header -->
```

Comment:
- [ ] This should be your page's toolbar, and you should use proper HTML tags to wrap the elements for better semantics and increase the SEO of your page. Kindly wrap this nav in a header tag. Typically the header tag is used as a page header; it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
- [ ] Kindly stick to the template you choose. In your design, to figure out some issues with alignment and missed icons. Please fix it :wrench:  

Example 

```
<header>
    <nav>
        <a> logo </a>
        <img src=" " alt=" humberger icon" />
    <nav>

<header>
```
***Template design***
![Screenshot from 2022-12-02 16-27-36](https://user-images.githubusercontent.com/65129228/205342022-72921b9d-fb86-4a27-b681-24806614e2ec.png)



***Your design***
![Screenshot from 2022-12-02 18-05-02](https://user-images.githubusercontent.com/65129228/205341214-37d24e20-10e2-43e2-8f9f-1db0e81d59e0.png)